### PR TITLE
Changed function to window.InitEasyPopup

### LINKED ISSUE #10: Migrate to webpack
Using https://github.com/phucbm/wellii

### PR DESCRIPTION
**Pull request by mr-cheffy**
this allows the user to reload all popups by calling `window.InitEasyPopup({})`. This was useful for me since I needed to reload popups when I override the elements in my body

### CODE REVIEW

File: src/easy-popup.js [hunk 1]
```diff
@@ -302,6 +302,8 @@
 
     // Get popup object by ID
     EasyPopup.get = id => window.EasyPopupData.get(id);
+    console.log("load")
```
Comment (phucbm) [inline src/easy-popup.js:305]: Could you remove this console log, please?

**Review by phucbm — CHANGES_REQUESTED** [review]
Thank you for making PR. There's one minor issue to fix tho.

File: src/easy-popup.js (hunk 1 above)
Comment (mr-cheffy) [reply src/easy-popup.js:305]: Sure

**Comment by phucbm** [comment]
Closed due to inactivity. We are drafting a new version of this and will include this usage.

**Pull request closed without merging**